cafes/views.py

```diff
@@ -3,12 +3,55 @@
 
 from django.http.response import JsonResponse
 from django.views         import View
-from django.db.models     import Count, Avg
+from django.db.models     import Count, Avg, Q
 
 from reviews.models import Review
 from cafes.models   import Cafe, Menu
 from ratings.models import StarRating
 from utils          import log_in_confirm
+from users.models   import User
+
+class UserCafeListView(View):
+    def get(self, request, user_id):
+        if not User.objects.filter(id=user_id).exists():
+            return JsonResponse({'MESSAGE' : 'USER_DOES_NOT_EXIST'}, status=400)
+
+        ordering = request.GET.get('ordering', None)
+        category = request.GET.get('category', None)
+
+        order    = {
+            'high_rating': '-avg_rating',
+            'low_rating' : 'avg_rating'
+        }
+
+        avg_ranking   = Cafe.objects.all().annotate(avg_rating=Avg('starrating__score'))\
+                                          .order_by(order.get(ordering, 'id'))
```

Comment:
한줄로 표현할 수 있을 것 같네요. 추가로 `Cafe.objects...` 호출 후 반환되는 값은 cafe 리스트입니다. 변수명 명확하게 만들어주세요.

```suggestion
        cafes = Cafe.objects.annotate(avg_rating=Avg('starrating__score')).order_by(order.get(ordering, 'id'))
```